cardano/base/templatetags/tags.py

```diff
@@ -13,3 +13,16 @@ def lookup_country_vouched(countries, key):
 def lookup_country_total(countries, key):
     q = max(countries, key=lambda d: (d['name'] == key))
     return q['total_diff']
+
+
+@register.filter
```

Comment:
On top of that, it's not a great idea to repeat the same multi-line string three times whith just a sub-string difference.